pkg/kubelet/container/runtime.go

```diff
@@ -556,6 +556,8 @@ type RuntimeStatus struct {
 	Conditions []RuntimeCondition
 	// Handlers is an array of current available handlers
 	Handlers []RuntimeHandler
+	// Features is the set of features implmented by the runtime
```

Comment:
typo: implemented

Reply:
Thanks! Fixed in f46ecf5648917e007f531e91865be2d70f2c9ea6